scripts/benchmark/command_stat_test.py

```diff
@@ -1,42 +1,46 @@
 #!/usr/bin/env python
 
 """
-This program tests if a command performs faster using independent t-test. At the top of the file,
-there are command placeholders which you need to define. You may change other constants
-if needed. Optionally, modify `performance_program`, a function used to simulate the
-work that is timed.
+This program tests if a command performs faster and with less memory. This is achieved
+using independent t-test. At the top of the file, there are command placeholders which
+you need to define. You may change other constants if needed. Optionally, modify
+`performance_program`, a function used to simulate the work that is benchmarked.
 
 The program will start the command defined in `ORIGINAL_COMMAND`,
-run `performance_program` `SAMPLE_SIZE` times, store these times,
+run `performance_program` `SAMPLE_SIZE` times, store these measurements,
 and repeat the same process for `IMPROVED_COMMAND`.
-The times shall than be statistically tested and the result printed.
+The measurements shall than be statistically tested and the result printed.
 """
 
 import subprocess
 import time
-from typing import List
+from typing import List, Tuple
 
+import psutil
 import requests
 from scipy.stats import ttest_ind
+from scipy.stats import describe
 
 DEVNET_PORT = "5050"
 DEVNET_URL = f"http://localhost:{DEVNET_PORT}"
 REQUEST_TIMEOUT = 2
 
 ORIGINAL_COMMAND: str = ...
```

Comment:
I'm suggesting adding some default values for `ORIGINAL_COMMAND` and `IMPROVED_COMMAND` so developers can run the script without reading the script itself, maybe:
```
ORIGINAL_COMMAND: str = f"cargo run --release -- --port {DEVNET_PORT}"
IMPROVED_COMMAND: str = f"cargo run --release -- --port {DEVNET_PORT} --lite-mode"
```
?

Later once they know that the script is running correctly they can tweak it for their demands.

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/464/commits/3d17e1ea1cc9caa2058eb6dcee00453d7afaa325